td-agent/yum/td-agent.spec.in

```diff
@@ -71,6 +71,13 @@ Requires(pre): /usr/bin/getent, /usr/sbin/adduser
 %description
 The stable distribution of Fluentd, called td-agent.
 
+%package devel
+Summary:  Development files for td-agent
+Requires: %{name} = %{version}-%{release}
+
+%description devel
+The stable distribution of Fluentd development package, called td-agent-devel.
```

Comment:
`, called td-agent-devel` should be removed as you do it in #78

Reply:
Ah, it should be fixed.